src/components/FilterTop.js

```diff
@@ -198,6 +201,36 @@ function Filter(props) {
                   </MenuItem>
                 ))}
               </TextField>
+              <TextField
+                select
+                label="Token Status"
+                value={
+                  tokenId === filterOptionAll
+                    ? filterOptionAll
+                    : tokenId === tokenizationStates.TOKENIZED
+                    ? tokenizationStates.TOKENIZED
+                    : tokenizationStates.NOT_TOKENIZED
+                }
+                onChange={(e) => {
+                  setTokenId(
```

Comment:
This can be simplified to `setTokenId(e.target.value)`.

Reply:
Heh whoops, thanks for catching this!